commands/uploader.go

```diff
@@ -332,13 +332,16 @@ func (c *uploadContext) Await() {
 }
 
 var (
+	githubHttps, _ = url.Parse("https://github.com")
+	githubSsh, _   = url.Parse("ssh://github.com")
```

Comment:
> If you add a `/` at the end of these, `strings.HasPrefix(u.Path, supported.Path)` will protect against urls like `https://github.com:123/...`.

Not totally sure on this one, since I think the `:123` port number would be a part of the `u.Host` and `u.Port()` member/function, but wouldn't appear in the `u.Path`.

> Also, does LFS add the `ssh://` prefix for git remotes like `git@github.com:...`?

Good question, the `*lfsapi.endpointGitFinder` looks at empty schemes in [`lfsapi/endpoint_finder.go:178-9`](https://github.com/git-lfs/git-lfs/blob/v2.0.2/lfsapi/endpoint_finder.go#L178-L179) and automatically adds the `ssh://` scheme. Some related work also in #2090.